pom.xml

```diff
@@ -48,8 +48,6 @@
     <!-- These properties are to be overwritten by the Maven modules -->
     <!-- Whether this module is an integration test module -->
     <gson.isTestModule>false</gson.isTestModule>
-    <!-- Whether this module is internal and currently not deployed -->
-    <gson.isInternalModule>${gson.isTestModule}</gson.isInternalModule>
```

Comment:
As mentioned in https://github.com/google/gson/pull/2900#discussion_r2337833744, `gson.isInternalModule` is not used anymore.